build_tools/cmake/test_riscv.sh

```diff
@@ -90,6 +90,7 @@ fi
 test_exclude_args+=(
   # TODO(#12703): Enable the test.
   "iree/tests/e2e/matmul/e2e_matmul_mmt4d_i8_small_llvm-cpu_local-task"
+  "iree/tests/e2e/tensor_ops/check_llvm-cpu_local-task_pack_dynamic_inner_tiles.mlir"
```

Comment:
Could you please include an issue or note when disabling tests? Why does this test need to be disabled? In general, disabling tests when checking in new code is indicative of a problem

Reply:
It is related to https://github.com/openxla/iree/issues/12703 . Its an LLVM -> RISC-V lowering error (does not seem to be an IREE error).

Reply:
Ah, I see: it's also related to the comment that's already there. I think that could be clearer in the comment (for instance, if it said "tests" instead of "test" :slightly_smiling_face:), but glad this is actually being tracked

Reply:
yeah, I checked that the github issue id is included, but I forgot to check if todo action items are up-to-date. I will review it carefully next time!